packages/ibm-products-styles/src/components/Datagrid/styles/_datagrid.scss

```diff
@@ -315,9 +390,15 @@
     white-space: initial;
   }
 
+  /* This section to be removed after support for slug dropped  - start */
   .#{$block-class}__defaultStringRenderer.#{$block-class}__defaultStringRenderer--slug {
     width: fit-content;
   }
+  /* This section to be removed after support for slug dropped  - end */
```

Comment:
same here too, this could be removed in this pr itself, and add corresponding aiLabel classname conditionally on presence of aiLabel || slug prop? can refer alex pr.